tests/analysis/normalization/test_normalization.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tests for the normalization modules
+
+Copyright © 2024 Pixelgen Technologies AB.
+"""
+
+from pathlib import Path
+
+import pandas as pd
+from pandas.testing import assert_frame_equal
+from pixelator.analysis.normalization import dsb_normalize
+
+DATA_ROOT = Path(__file__).parents[2] / "data"
+
+
+def test_dsb_normalize():
```

Comment:
Perhaps we should also add a test where `isotype_controls=None` to make sure we cover that code path.